run.sh

```diff
@@ -16,14 +16,20 @@ while [ -n "$1" ]; do
 done
 
 SDC_HOME=${SDC_HOME:-$(mktemp -d)}
+DB_HOME=$SDC_HOME/data
```

Comment:
This logic should be kept inside the app. There should only be a single injection of `SDC_HOME`, and the app should figure everything else out from there.

Reply:
This helps to make sure that I am creating the directory structure properly, as I need that directory structure to create the DB file.

Reply:
All of this logic is inside the app. Doing it in two places runs the risk of us relying on something we shouldn't.